### ISSUE TITLE
Code? License?

### ISSUE THREAD

**Issue by oiisamiio** [elegooofficial/ELEGOO-Smart-Robot-Car-Kit-V4.0#2]
I want fork Robot Car v4.0. What about a licensing model?

Why was a zip filed instead of the code?

Regards

**Comment by petres** [comment]
I had the same question. As they ship the GPL licensed library `NewPing` in the zip file `SmartRobotCarV4.0_V0_20210104.zip`, I assume forking shouldn't be a problem.

**Comment by alx77** [comment]
Yes, I also want to know about licensing, I made a fork for Arduino Zero (SAMD21), seems like these boards are (almost) replaceable with Uno. I also added support of ELRS RC Receiver. How can I publish it? Also I'm looking for the source code of Elegookit app, where can I find it?

**Comment by petres** [comment]
It seems that ELEGOO doesn't care at all to give us here feedback. I also looked for the source code of the App, but without success. I have implemented a web backend/frontend to replace teh app, still not perfect, but it works. But you would need some kind of server, a raspy should work. https://github.com/petres/robot-car/tree/main/web
